README.md

```diff
@@ -189,9 +190,10 @@ client.on('message', (message) => {
 -   **options.newPrize**: the new prize.  
 -   **options.addTime**: the number of milliseconds to add to the giveaway duration.
 -   **options.setEndTimestamp**: the timestamp of the new end date (for example, for the giveaway to be ended in 1 hour, set it to `Date.now() + 60000`).
--   **options.newMessages**: the new giveaway messages
--   **options.newExtraData**: the new extra data value for the giveaway
+-   **options.newMessages**: the new giveaway messages.
+-   **options.newExtraData**: the new extra data value for the giveaway.
 -   **options.newBonusEntries**: the new BonusEntry objects (for example, to change the amount of entries).
+-   **options.newisDrop**: wheter to convert a drop to a normal giveaway or vice versa.
```

Comment:
```suggestion
-   **options.newIsDrop**: wheter to convert a drop to a normal giveaway or vice versa.
```
Bro, [CamelCase](https://techterms.com/definition/camelcase) as said before, and you already approved `newIsDrop` in the constant, giveaway and types file...

Reply:
Oh yes so sorry about that